packages/hawtio/src/plugins/jmx/JmxContent.tsx

```diff
@@ -63,20 +78,22 @@ export const JmxContent: React.FunctionComponent = () => {
       <PageSection>
         {node.objectName &&
           <React.Fragment>
-            {mbeanRoutes}
-            <Route key='root' exact path='/jmx' component={Attributes} />
+            <Routes>
+              {mbeanRoutes}
+              <Route key="root" path="/*" element={<Navigate to={"attributes"}/>}/>
```

Comment:
with this path setting, when accessing http://localhost:3000/hawtio/?con=springboot from Connect it redirects to http://localhost:3000/hawtio/jmx/*?con=springboot (note `*`). Why can't it be `/`?